packages/amplify_core/lib/src/amplify_class.dart

```diff
@@ -103,6 +103,8 @@ abstract class AmplifyClass {
       );
     }
 
+    AmplifyLogger().registerPlugin(AnsiPrettyPrinter());
```

Comment:
I wanted to see what this looks like by default, so I added the following log statements

```dart
final logger = AmplifyLogger()
      ..registerPlugin(AnsiPrettyPrinter())
      ..logLevel = LogLevel.verbose;

    logger.verbose('verbose message');
    logger.debug('debug message');
    logger.info('info message');
    logger.warn('warn message');
    logger.error('error message');
```

And this is what I see (In VS Code and in a zsh and bash shell). Im I doing something wrong?

<img width="718" alt="Screen Shot 2022-07-12 at 10 42 50 AM" src="https://user-images.githubusercontent.com/20613561/178519418-1fa8c501-c648-4380-85c5-041b36260f1d.png">

Reply:
Thanks for catching that Jordan.  That's a bummer it won't work there.  I created a SimplePrinter and switched it to be the new default.

Reply:
Hm. Idk what I did differently, but it seems to be working in VS code debugger now.

I would reduce the default formatting though. It is a lot IMO. Whatever is written in the yellow background is not legible. Definitely would not pass the contrast ratio on an accessibility test 🙂 .

<img width="347" alt="Screen Shot 2022-07-12 at 6 03 45 PM" src="https://user-images.githubusercontent.com/20613561/178603325-1099e420-4ad9-4648-b1f5-4189abd2bdab.png">

Reply:
We also need to make sure this looks reasonable in all themes if we support it. It looks somewhat odd in light mode.

<img width="347" alt="Screen Shot 2022-07-12 at 6 11 18 PM" src="https://user-images.githubusercontent.com/20613561/178604257-faf1556b-0c43-46af-bc03-862f68858d5d.png">